### PR TITLE
issue1744 serialOf on factus interface

### PR DESCRIPTION
**Pull request by uweschaefer**

### CODE REVIEW

**Comment by uweschaefer** [comment]
closes #1744

File: factcast-factus/src/test/java/org/factcast/factus/projector/ProjectorImplTest.java [hunk 1]
```diff
@@ -257,7 +257,7 @@ void invalidPostprocessReturnsNull() {
     void invalidPostprocessReturnsEmptyList() {
       // INIT
       ProjectorImpl<Projection> underTest =
-          new ProjectorImpl<>(eventSerializer, new PostProcessingProjection(Lists.emptyList()));
+          new ProjectorImpl<>(eventSerializer, new PostProcessingProjection(Collections.emptyList()));
```
Comment (otbe) [inline factcast-factus/src/test/java/org/factcast/factus/projector/ProjectorImplTest.java:260]: Seems to be unrelated 🥸

Comment (uweschaefer) [reply factcast-factus/src/test/java/org/factcast/factus/projector/ProjectorImplTest.java:260]: removed compiler warning (deprecation)
